docs/api.md

```diff
@@ -51,13 +51,13 @@ An array of functions to be called after Electron has been extracted to a tempor
 
 When `true`, sets both [`arch`](#arch) and [`platform`](#platform) to `all`.
 
-##### `app-copyright`
+##### `appCopyright`
```

Comment:
@malept Should I place the deprecated names right underneath the new one, or should I group them at the bottom?